apps/api/src/domain/common/events/interfaces/coscrad-event-publisher.interface.ts

```diff
@@ -7,5 +7,5 @@ export interface ICoscradEventPublisher {
     publish(eventsToPublish: ICoscradEvent | ICoscradEvent[]): void;
 
     // TODO Should we have a separate event bus? What does this look like with a messaging queue?
```

Comment:
Can we reword this comment?

Reply:
I removed this comment for now. We know already that this is the direction we intend to go.